### PR TITLE
Feature/dss12 sc 181423 multi sheet excel export apply a formatting

### PR DESCRIPTION
**Pull request by Ellana42**
Added a styled header to the exported spreadsheets - similar to the style of DSS native excel export. The columns are also auto-sized to fit the header text.

### CODE REVIEW

**Comment by Ellana42** [comment]
[sc-181422]

**Review by MayeulRousselet — CHANGES_REQUESTED** [review]
LGTM and tested, very solid work!
Only a small comment todo 
Tested:
- On a empty dataset: the same error as in the previous version is raised
- 2 datasets as input: 2 sheets are generated
- with an empty colum: header is added
- With 1M lines

**Review by MayeulRousselet — APPROVED** [review]

**Review by alexbourret — APPROVED** [review]
👍 All good !

File: python-lib/xlsx_writer.py [hunk 1]
```diff
@@ -7,11 +7,73 @@
...
+                  font_color : str = WHITE, 
+                  background_color : str = DATAIKU_TEAL,
+                  bold : bool = True
+                 ):
+    """
+    Style header of the worksheet
+    """
+
+    if worksheet.min_column < 1:
+        logger.warn(f"No header row for worksheet {worksheet}. Styling skipped.")
```
[the remark this answers is not in the document]
Comment (Ellana42) [reply python-lib/xlsx_writer.py:40]: yes, I checked

**Pull request merged**